tests/Unit/CssInlinerTest.php

```diff
@@ -1606,13 +1606,18 @@ public function orderedRulesAndSurroundingCssDataProvider()
                 foreach ($possibleSurroundingCss as $descriptionAfter => $cssAfter) {
                     // every combination would be a ridiculous c.1000 datasets - choose a select few
                     // test all possible CSS before once
-                    if (($cssBetween === '' && $cssAfter === '')
+                    if (
```

Comment:
Could this be resolved by merely adding a newline after `if (`...?

Reply:
I'll test this.

This is the relevant section from the standard: https://www.php-fig.org/psr/psr-12/#51-if-elseif-else

Reply:
Unfortunately, this did not work. :-(

Reply:
We could use introduce some local variables (in a separate PR) to make the condition less enormous, though.

Reply:
> Unfortunately, this did not work. :-(

OK.

> We could use introduce some local variables (in a separate PR) to make the condition less enormous, though.

That would help make it more readable and perhaps also avoid the need for the comments within the `if` expression.